_maps/configs/nanotrasen_harrier.json

```diff
@@ -0,0 +1,75 @@
+{
```

Comment:
Add `"faction": "/datum/faction/nt"`, somewhere, or it's gonna show as indie on the spawning menu.